### PR TITLE
link for collection details updated

### PR DESCRIPTION
**Pull request by martin-raden**

### CODE REVIEW

**Comment by martin-raden** [comment]
ping @nsoranzo

**Comment by martin-raden** [comment]
fixes  #962

File: docs/_writing_collections.rst [hunk 1]
```diff
@@ -18,8 +18,9 @@ types - ``list`` and ``paired``.
   applications. Each ``paired`` collection consists of a ``forward`` and
   ``reverse`` dataset.
 
-.. note:: Read more about creating and managing collections on the
-  `Galaxy Wiki <https://wiki.galaxyproject.org/Histories#Dataset_Collections>`__.
+.. note:: Read more about
+  `creating and managing collections <https://wiki.galaxyproject.org/Histories#Dataset_Collections>`_
```
Comment (nsoranzo) [inline docs/_writing_collections.rst]:
```suggestion
  `creating and managing collections <https://galaxyproject.org/tutorials/collections/>`_
```

**Comment by nsoranzo** [comment]
Thanks @martin-raden !

**Pull request merged**